en/docs/insights/derive-insights.md

```diff
@@ -0,0 +1,364 @@
+
+![Error statistics](../assets/img/insights/error-page-full.png){.cInlineImage-full}
+
+You can filter the information displayed in the widgets as follows:
+
+| **Filtering Option** | **Description** |
+|--------------------|-------------------------------------------------------------------------------------------------|
+| **By API** | In the **API** field, select the APIs for which you want to view analytics. By default, all APIs are selected. You can select one or multiple APIs from this selector and view the aggregated result. |
+| **By Category** | The **Category** field is displayed when you click **Category** in the toggle switch in the upper right corner of the page. In this field, you can select one or all of the following error categories:<br/>**Authentication**: Any kind of authentication error falls into this category including expired/missing/invalid credentials.<br/>**Target Connectivity**: Any kind of API back-end error falls into this category including connection time-outs/other back-end errors (for example, 4xx and 5xx status codes).<br/>**Throttling**: Any kind of API back-end error falls into this category including application throttling/subscription throttling.<br/>**Other**: All other errors fall into this category including mediation errors/resource not found errors. |
+| **By Status Code** | The **Status Code** field is displayed when you click **Status Code** in the toggle switch in the upper right corner of the page. In this field, you can select all or any available HTTP status code category (4xx, 5xx, etc.). These status codes represent the response status of the API Gateway. |
+
+#### Category mode
+
+The following widgets are available for monitoring errors when you have selected the category mode via the toggle switch in the upper-right corner of the page.
+
+- **Errors by Category** (graph)
+
+    This graph shows the error category distribution over time for the selected period. Apply the required filters as explained above to select the APIs and the error categories to which this content applies. When you select multiple APIs, the error count is grouped by category.
+
+    ![Errors by category graph](../assets/img/insights/error-category-graph.png){.cInlineImage-full}
+    
+- **Errors by Category** (table)
+
+    This table provides further information about the errors such as application details and the error reason. For some authentication errors, the application name is not available. You can use this table to get more concrete information about the errors related to your APIs and then start the problem isolation process.
+ 
+    ![Errors by category table](../assets/img/insights/error-category-table.png){.cInlineImage-full}
+    
+#### Status Code mode
+    
+The following widgets are available for monitoring errors when you have selected the status code mode via the toggle switch in the upper-right corner of the page.
```

Comment:
here too since we are talking about an input, shall we make "status code mode"  bold?

Reply:
There is no input value called "status code mode". It is a switch.
<img width="275" alt="Screenshot 2021-09-06 at 14 22 55" src="https://user-images.githubusercontent.com/26239861/132189235-6c7139cb-f70f-4789-8e3b-929f1e1db115.png">

Reply:
It doesn't seem correct to say toggle switch. Shall we rephrase the sentence accordingly? I guess we can replace "when you have selected" with "when you select" too.

Reply:
Changed this. Please check.